app/messages.py

```diff
@@ -152,73 +147,64 @@
+    "message": "Mentorship relation minimum duration is 4 week."
 }
 CANT_ACCEPT_MENTOR_REQ_SENT_BY_USER = {
-    "message": "You cannot accept a" " mentorship request sent by yourself."
+    "message": "You cannot accept a mentorship request sent by yourself."
 }
 CANT_ACCEPT_UNINVOLVED_MENTOR_RELATION = {
-    "message": "You cannot accept a"
-    " mentorship relation where you are"
-    " not involved."
+    "message": "You cannot accept a mentorship relation where you are not involved."
 }
 USER_CANT_REJECT_REQUEST_SENT_BY_USER = {
-    "message": "You cannot reject a" " mentorship request sent by yourself."
+    "message": "You cannot reject a mentorship request sent by yourself."
 }
 CANT_REJECT_UNINVOLVED_RELATION_REQUEST = {
-    "message": "You cannot reject a"
-    " mentorship relation where you are"
-    " not involved."
+    "message": "You cannot reject a mentorship relation where you are not involved."
 }
 CANT_CANCEL_UNINVOLVED_REQUEST = {
-    "message": "You cannot cancel a mentorship" " relation where you are not involved."
+    "message": "You cannot cancel a mentorship relation where you are not involved."
 }
 CANT_DELETE_UNINVOLVED_REQUEST = {
-    "message": "You cannot delete a mentorship" " request that you did not create."
+    "message": "You cannot delete a mentorship request that you did not create."
 }
 NOT_IN_MENTORED_RELATION_CURRENTLY = {
-    "message": "You are not in a current" " mentorship relation."
+    "message": "You are not in a current mentorship relation."
 }
 USER_IS_INVOLVED_IN_A_MENTORSHIP_RELATION = {
-    "message": "You are currently" " involved in a" " mentorship relation."
+    "message": "You are currently involved in a mentorship relation."
 }
 USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION = {
-    "message": "You are not involved" " in this mentorship relation."
+    "message": "You are not involved in this mentorship relation."
 }
 USER_USES_A_USERNAME_THAT_ALREADY_EXISTS = {
-    "message": "A user with that" " username already exists."
+    "message": "A user with that username already exists."
 }
 USER_USES_AN_EMAIL_ID_THAT_ALREADY_EXISTS = {
-    "message": "A user with that " "email already exists."
+    "message": "A user with that email already exists."
 }
 USER_IS_NOT_REGISTERED_IN_THE_SYSTEM = {
-    "message": "You are not registered in" "the system."
+    "message": "You are not registered in the system."
 }
 NAME_LENGTH_GREATER_THAN_MAX_LIMIT = {
-    "message": "The {field_name} field has"
-    " to be no more than {max_limit}"
-    " characters."
+    "message": "The {field_name} field has to be no more than {max_limit} characters."
 }
-NAME_LENGTH_LESSER_THAN_MAX_LIMIT = {
-    "message": "The {field_name} field has to"
-    " be at least than {min_limit} characters"
-    " and no more than {max_limit}"
+NAME_LENGTH_NOT_VALID = {
+    "message": "The {field_name} must have at least {min_limit} characters and at most {max_limit}"
```

Comment:
We already have NAME_LENGTH_LESSER_THAN_MAX_LIMIT above. 
```suggestion
NAME_LENGTH_LESSER_THAN_MIN_LIMIT = {
    "message": "The {field_name} has to be at least {min_limit} characters."
```